puzzle_generator/bytes_utils.py

```diff
@@ -5,14 +5,13 @@
 
 
 def int_to_bytes(in_value: int) -> bytes:
-    assert in_value >= 0  # nosec B101
+    if in_value < 0:
+        raise ValueError("in_value must be a positve integer")
     number_of_bytes = in_value.bit_length() // 8
     if 8 * number_of_bytes < in_value.bit_length():
         number_of_bytes += 1
```

Comment:
Could you please move this into a separate function like `byte_length`. This will allow to test it properly.